cmd/githubrunnerscalesetlistener/main.go

```diff
@@ -179,123 +154,103 @@ func main() {
 		os.Exit(1)
 	}
 
-	runnerScaleSetSession, err := actionsServiceClient.CreateMessageSession(ctx, runnerScaleSet.Id, hostName)
-	if err != nil {
+	retries := 3
+	var lastErr error
+	var runnerScaleSetSession *github.RunnerScaleSetSession
+	for i := 0; i < retries; i++ {
+		runnerScaleSetSession, err = actionsServiceClient.CreateMessageSession(ctx, runnerScaleSet.Id, hostName)
```

Comment:
NIT: If we don't care about what is in err variable, should we just use `lastErr`? We are assigning to `lastErr` on the next line

Reply:
Fixed